library/include/borealis/core/application.hpp

```diff
@@ -210,6 +210,11 @@ class Application
      */
     static std::string getLocale();
 
+    /**
+     * View that was recognized as target for touch events.
+     */
+    inline static View* firstResponder; 
```

Comment:
I use it inside View to interrupt other recognisers if one change its state from UNSURE to START.
I could throw it as argument from Application but I don't think it's an elegant solution.

Reply:
Do you think it would be possible to have this inside FrameContext instead, and toss the context around where you need it? Would it make touch eventss more difficult to use?

Reply:
As I can see FrameContext constructs after all input logic done, do you want to move it inside Application::frame() ?
I think it will not be a problem, I'll just add a firstResponder argument in gestureRecognizerRequest method, and will toss it from Application where we construct FrameContext, cause I don't think that I need full FrameContext inside touch logic. 
Just notify me when you'll done these changes, I'll adopt my code to your solution.

Reply:
Moved first responder from public header.